### PR TITLE
Update bootloader to the latest stable release and use full SDRAM init in MSD / RAMDISK mode.

### PR DESCRIPTION
**Pull request by timg236**
4fcf451 eeprom-erase: Add option to erase the SPI EEPROM
6e51392 secure-boot: Specify the minimum version for secure-boot mode.
0eb1b63 Update EEPROM to latest stable release 2022-01-25

### CODE REVIEW

**Comment by timg236** [comment]
@peterharperuk

File: secure-boot-recovery/README.md [hunk 1]
```diff
@@ -72,9 +72,11 @@ onwards:
 * The EEPROM configuration file must be signed with the customer private key.
 * It is not possible to install an old version of the bootloader that does
   support secure boot.
-* **It is NOT possible to use a different private key to signed the OS images**
+* This option requires EEPROM version 2022-01-06 or newer.
+* BETA bootloader releases are not signed with the ROM secure boot key and will
+  not boot on a system where `revoke_devkey` has been set.
 
-**WARNING: THESE OPTIONS PERMANENTLY THE BCM2711 CHIP AND ARE IRREVERSIBLE.**
+**WARNING: Modifying OTP is irreversible and once set and it is not possible to unlock secure-boot mode or use a private key.**
```
Comment (andrum993) [inline secure-boot-recovery/README.md:79]: One too many `and` there 😉

**Comment by peterharperuk** [comment]
Looks good

File: secure-boot-recovery/README.md (hunk 1 above)
Comment (timg236) [reply secure-boot-recovery/README.md]: Fixed

**Pull request merged**